redocly/scripts/combine-yamls.py

```diff
@@ -0,0 +1,136 @@
+import yaml
+import re
+import os
+
+# Paths to the API spec files to combine
+file_paths = {
+    "authentication": "../../res/authentication-apis.yaml",
+    "data_in": "../../res/data-in-apis.yaml",
+    "data_out": "../../res/data-out-apis.yaml",
+    "administration": "../../res/administration-apis.yaml",
+    "analytic_model": "../../res/analytic-model-apis.yaml",
+    "skills_intelligence": "../../res/skills-intelligence-engine.yaml",
+    "compensation_benchmarks": "../../res/compensation-benchmarks.yaml"
+}
+
+# List of tags to exclude from the final OpenAPI spec
+excluded_tags = {
+    "administration": ["TenantsV1", "TenantsV2"]
```

Comment:
I know we've suppressed the tenant APIs from the regular dev docs, but they're still needed for embedded docs. How does embedded get them if they're suppressed here?

Reply:
I just synced with the docs team and we have decided to not exclude these anymore, but I will keep the variable here in case there needs to be added excluded specs.